app/domain/authentication/security/validate_webservice_exists.rb

```diff
@@ -1,12 +1,9 @@
 # frozen_string_literal: true
 
-require 'authentication/validate_account_exists'
-
 module Authentication
 
   module Security
 
-    Err ||= Errors::Authentication::Security
     # Possible Errors Raised:
     # AccountNotDefined, WebserviceNotFound
```

Comment:
Doesn't seem like `AccountNotDefined`  is used

Reply:
Neither is `WebserviceNotFound`

Reply:
removed